README.md

```diff
@@ -90,6 +90,8 @@ for (pid, process) in sys.get_processes() {
 }
 ```
 
+Sysinfo supports parallel retrieval of system processes with the `multithread` cargo feature, which is enabled by default.
```

Comment:
What do you think of:

```
By default, `sysinfo` is using multithreading by default to get better performance. However, it can increase the memory usage on some platforms (macOS for example). If you want to disable it, use `--no-default-features` to disable the `multithread` cargo feature which is enabled by default.
```

Maybe a bit too verbose? Well, I'm not an english native so I'm open to suggestions. :)

Reply:
I like it, more info and a hint on how to disable it!  Just pushed a new version with a modified comment:

```
By default, `sysinfo` uses multiple threads. However, this can increase the memory usage on some platforms (macOS for example).  The behavior can be disabled by setting `default-features = false` in `Cargo.toml` (which disables the `multithread` cargo feature).
```

Reply:
There is still the old comment, is it voluntary?

Reply:
Oh sorry, my mistake.  I rebased and removed the original comment.